### DISCUSSION TITLE
Forms text locked - cannot edit

### DISCUSSION THREAD

**Discussion by CJStevens17** [sheltermanager/asm3#1475] [Q&A / Help]
I am new but have looked everywhere to find out how to change the verbiage on a form. I can change the input fields but not the boilerplate text. Thanks

**Comment by Tetzor** [comment]
Hi, your template is using an image of an externally created document. To edit text in the image you will need to edit the original document and upload a new image. I have sent you an email with the steps on how to do this.

Jon
